Automattic/wp-api-console
# Add the share button

Every time we need to provide instructions to test an endpoint using the Console, we also need to provide all the information related to the request so the person reviewing it can test it properly. This led me to think of a way to share an URL with all the information the test need without having to select anything.

With this PR, we can now share a URL like:
`https://developer.wordpress.com/docs/api/console?api=WP+REST+API&version=wpcom%2Fv2&path=%2Fsites%2Fvlorran999.wordpress.com%2Fposts%2F1&method=POST`

And this will automatically fill in the information as shown below:

![Screen Shot 2023-07-28 at 14 14 36](https://github.com/Automattic/wp-api-console/assets/1234758/c5a6f6f4-4b1a-4c6c-8987-89f26928969d)

#### Testing

* Apply this PR
* Navigate to the `http://localhost:3000/`
* Fill in some information about a request
* Click on the copy button:
![Screen Shot 2023-07-28 at 14 16 26](https://github.com/Automattic/wp-api-console/assets/1234758/f8e3ff29-2ebe-48a1-b2c4-2c3f8b8acadd)
* Paste the link in a new tab
* You should see the exact same options you defined earlier
